manifests/params.pp

```diff
@@ -20,7 +20,11 @@
         'Ubuntu' => if versioncmp($facts['os']['release']['major'], '22.04') >= 0 { '/usr/bin/named-checkconf' } else { '/usr/sbin/named-checkconf' },
         default  => '/usr/sbin/named-checkconf',
       }
-      $sysconfig_file     = '/etc/default/bind9'
+      $sysconfig_file     = $facts['os']['name'] ? {
+        'Debian' => if versioncmp($facts['os']['release']['major'], '10') >= 0 { '/etc/default/bind9' } else { '/etc/default/named' },
```

Comment:
We only support Debian  >= 10 so should this be simple:
```suggestion
        'Debian' => '/etc/default/bind9',
```

But it's actually not what reflects reality. I think you meant this:
```suggestion
        'Debian' => if versioncmp($facts['os']['release']['major'], '11') >= 0 { '/etc/default/named' } else { '/etc/default/bind9' },
```

Reply:
thanks for the simplification, I pushed a change.